nasa/harmony
# DAS-2255: Adds config and environment for harmony-smap-l2-gridder

## Jira Issue ID

[DAS-2255](https://bugs.earthdata.nasa.gov/browse/DAS-2255)

## Description

Add Harmony SMAP L2 Gridding Service configuration to Harmony repository.


## Local Test Steps


1. Pull the SMAP docker image from ghcr.io
    `docker pull ghcr.io/nasa/harmony-smap-l2-gridding-service:latest`

1. Update your local Harmony-In-A-Box configuration to include smap-l2-gridder in the `LOCALLY_DEPLOYED_SERVICES`
    `LOCALLY_DEPLOYED_SERVICES=<[blah,blah,etc]>,smap-l2-gridder`

1. start your Harmony-In-A-Box
    `./bin/bootstrap-harmony`

1. Jump onto a NASA VPN.

1. Make a request for a configured collection
     http://localhost:3000/C1263071064-NSIDC_CUAT/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?forceAsync=true&outputcrs=EPSG%3A4326&format=application%2Fx-netcdf4&maxResults=1

1. Open your workflow-ui and see that it is running the smap-l2-gridder.

1. Visit the job page and download the asset `SMAP_L2_SM_P_E_46849_A_20231108T210834_R19240_003_regridded.nc`

1. Open the file and see that it looks and behaves as expected with xarray or panoply.


## PR Acceptance Checklist
* [N/A] Tests added/updated (if needed) and passing
* [N/A] Documentation updated (if needed)
